platform/src/main/java/org/stellar/anchor/platform/config/PropertySep38Config.java

```diff
@@ -1,10 +1,26 @@
 package org.stellar.anchor.platform.config;
 
 import lombok.Data;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
 import org.stellar.anchor.config.Sep38Config;
 
 @Data
-public class PropertySep38Config implements Sep38Config {
+public class PropertySep38Config implements Sep38Config, Validator {
   boolean enabled = false;
   String quoteIntegrationEndPoint;
+
+  @Override
+  public boolean supports(Class<?> clazz) {
+    return Sep38Config.class.isAssignableFrom(clazz);
+  }
+
+  @Override
+  public void validate(Object target, Errors errors) {
+    Sep38Config config = (Sep38Config) target;
+
+    ValidationUtils.rejectIfEmpty(
+        errors, "quoteIntegrationEndPoint", "empty-quoteIntegrationEndPoint");
```

Comment:
Yes, I should also check for whitespace.

I've also had the thought about validating URLS. I think it's a good idea, but I also wasn't sure if we wanted to create a hard dependency on a valid endpoint. This would cause make dev envs (for us and our partners) and unit tests less convenient.

One option might be to leave a warning instead of a failure, or only do this check when some flag for strict validation checks. What do you think?

Reply:
> I've also had the thought about validating urls. I think it's a good idea, but I also wasn't sure if we wanted to create a hard dependency on having valid endpoints. This would cause make dev envs (for us and our partners) and unit tests less convenient.

Great point!

> One option might be to leave a warning instead of a failure, or only do this check when some flag for strict validation checks. What do you think?

I like that idea!